src/localization/formatted-message.tsx

```diff
@@ -21,7 +21,17 @@
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 
-const EnhancedFormattedMessage = props => (
+type FormattedMessageType = {
```

Comment:
For react components... Replace Type with Props? 
```suggestion
type FormattedMessageProps = {
```

Reply:
agreed, use `EnhancedFormattedMessageProps` to match the component name